### DISCUSSION TITLE
Flow export

### DISCUSSION THREAD

**Discussion by k-ilias** [akvorado/akvorado#50] [Q&A]
Hello,
Sorry, I don't understand, should I create a file with this configuration or is it somewhere?
![image](https://user-images.githubusercontent.com/44038537/180814248-a8f5038d-9b0c-4d5c-963f-d64c5e4763d0.png)

**Comment by vincentbernat** [comment]
It's `akvorado.yaml` at the root of the repository. And it's already present. If you use the `docker-compose` setup, be sure to look at the intro and troubleshooting section of the documentation.

**Reply by k-ilias** [reply]
user@test-server:~/akvorado/bin$ ./akvorado inlet -C
Error: accepts 1 arg(s), received 0
How can i check conf?

**Reply by k-ilias** [reply]
and i dont see flow, but i see connection
![image](https://user-images.githubusercontent.com/44038537/180996103-164c9d42-9c5c-43ac-a630-72f0ee621c06.png)

**Reply by vincentbernat** [reply]
When using docker-compose, use `127.0.0.1:8080` instead of `akvorado` when invoking curl.

**Comment by k-ilias** [comment]
curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1
That's right?

**Reply by vincentbernat** [reply]
Yes. It should either hang or return a flow. From there, follow the troubleshooting documentation.

**Reply by k-ilias** [reply]
sorry for my inexperience and misunderstanding
![image](https://user-images.githubusercontent.com/44038537/181001615-e1d26023-154c-44fa-8ff8-a4bdefacc5e5.png)

**Reply by k-ilias** [reply]
404 page not found, how fix it?

**Reply by georgetasioulis** [reply]
@k-ilias I would recommend you to avoid copy/pasting commands blindly - and to be honest, based on your screenshot you should probably assign your sysadmin the task to set this up for you, or better yet wait a couple of months for a beta/stable release. 
As @vincentbernat has already mentioned, at it's current state, akvorado should be treated as an alpha version.

That said, if curl to 127.0.0.1:8080 returns a 404 it's quite possible something else is running on port 8080. Did you launch this on a fresh server with only docker installed?

**Reply by k-ilias** [reply]
i understand, excuse for troubling. I check  my error.
